src/ga/population.rs

```diff
@@ -139,6 +141,46 @@ impl PopulationGenerator<Vec<bool>> for BitStrings {
   }
 }
 
+/// Implements [PopulationGenerator] trait. Can be used with genetic algorithm.
+///
+/// Generates random permutations of given vector.
+/// Permutations can be repeated
+pub struct RandomPermutations<GeneT: Copy> {
+  genes: Vec<GeneT>,
+}
```

Comment:
I guess we should also add a possibility to specify an RNG, as it was done here: https://github.com/ecrs-org/ecrs/pull/205

Maybe we should actually create separate issue for this.